micall/tests/test_consensus_aligner.py

```diff
@@ -816,11 +835,16 @@ def test_count_coord_concordance():
     aligner.coordinate_name = 'test-region'
     aligner.alignments = [AlignmentWrapper(r_st=0, r_en=27, q_st=0, q_en=27, cigar=[[27, CigarActions.MATCH]])]
 
-    expected_concordance_list = [None]*10 + [1.0, 1.0, 1.0, 0.95, 0.9, 0.9, 0.9, 0.9] + [None]*9
+    # At the end of the consensus, the size of the averaging window for the concordance decreases from 20 to 11.
+    # The concordance therefore decreases from 18/20 to 9/11
+    expected_concordance_list = [1.0]*10 + [1.0, 1.0, 1.0, 0.95, 0.9, 0.9, 0.9, 0.9] +\
+                                [17/19, 16/18, 15/17, 14/16, 13/15, 12/14, 11/13, 10/12, 9/11]
```

Comment:
I think it would be easier to understand if the values are either all fractions or all decimals.